src/lib/y2network/widgets/bond_slave.rb

```diff
@@ -69,10 +68,8 @@ def help
 
       # Default function to init the value of slave devices box for bonding.
       def init
-        items = slave_items_from(
-          Yast::LanItems.GetBondableInterfaces(Yast::LanItems.GetCurrentName),
-          @settings["SLAVES"]
-        )
+        # TODO: use def items, but problem now is that slave_items returns term and not array
+        items = slave_items_from(@settings.bondable_interfaces, @settings["SLAVES"])
 
         # reorder the items
         l1, l2 = items.partition { |t| @settings["SLAVES"].include? t[0][0] }
```

Comment:
it seems `@settings["SLAVES"]` is not properly initialized, so faced a nil.include? exception